paper/paper.md

```diff
@@ -82,7 +82,7 @@ for data in dataloader:
 from torchsurv.loss import weibull
 
 # PyTorch model outputs TWO Weibull parameters per observation
-my_model = MyPyTorcWeibullhModel() 
+my_model = MyPyTorchWeibullhModel() 
```

Comment:
@tcoroller could you fix the typo MyPyTorchWeibullModel (without the h). Sorry about that

Reply:
Done